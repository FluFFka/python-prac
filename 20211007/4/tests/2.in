"str(x)", "'werwerwerwerwer'", "x*2+y*3+'123'"
"hhreinkj(q123)+j"
