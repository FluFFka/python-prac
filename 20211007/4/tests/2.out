hhreinkj(q123)+jhhreinkj(q123)+jwerwerwerwerwerwerwerwerwerwerwerwerwerwerwer123
